src/server/cluster/cluster_slot_migration.cc

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+#include "server/cluster/cluster_slot_migration.h"
+
+#include <absl/flags/flag.h>
+
+#include "base/logging.h"
+#include "server/error.h"
+#include "server/main_service.h"
+
+ABSL_FLAG(int, source_connect_timeout_ms, 20000,
```

Comment:
I know this is a draft PR , but make sure you dont use "master" and "replication" in here. Instead use souce/target node and migration